scripts/docker-integration-tests/m3aggregator.Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM alpine:latest AS builder
```

Comment:
Can we open an issue to track pending tasks and include the documentation, but also adding support for this to m3_stack so we can play around with it locally easily

Reply:
Also in the issue we should include that we will need to create a Grafana dashboard

Reply:
We already have it in M3 stack actually, can add a dashboard request to an issue for follow ups, let’s do that.

Reply:
Opened issue: https://github.com/m3db/m3/issues/1507